src/features/organizations/components/ProceduralColorIcon/index.tsx

```diff
@@ -1,20 +1,34 @@
-import { Box } from '@mui/material';
 import { FC } from 'react';
 import { GroupWork } from '@mui/icons-material';
+import { makeStyles } from '@mui/styles';
 import randomSeed from 'random-seed';
+import { Box, Theme } from '@mui/material';
+
+interface StyleProps {
+  rgbAverage: number;
+}
+
+const useStyles = makeStyles<Theme, StyleProps>((theme) => ({
```

Comment:
Maybe could do the cool `r`, `g`, `b` maths inside here. Then return this object and classes for the background as well. Then it could just take the randomseed and do all the maths in the makeStyles callback

Reply:
I like this, but also think it's a very minor thing, so I'm inclined to just go ahead and merge anyway.